guide/resources/introduction.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: guide
+title: Resources
+description: Resources to help you build better bitcoin products.
+nav_order: 8
+has_children: true
+permalink: /guide/resources

```

Comment:
```suggestion
permalink: /guide/resources/
```

Jekyll is a bit annoying with the `/` at the end of links. Specifically, rake test will see URLs with and without the last `/` as different and throw errors. So we need to always add it.